plugin/core/handlers.py

```diff
@@ -17,9 +17,19 @@ def config(self) -> ClientConfig:
 
     @classmethod
     def instantiate_all(cls) -> 'List[LanguageHandler]':
-        return list(
-            instantiate(c) for c in cls.__subclasses__()
-            if issubclass(c, LanguageHandler))
+        subclasses = []  # type: List[Type[LanguageHandler]]
+        cls.get_final_subclasses(cls.__subclasses__(), subclasses)
+        return list(instantiate(c) for c in subclasses)
```

Comment:
Perhaps this is also a good time to put this under a try ... except block because right now, if one instantiation of a faulty language handler fails, *all* language handlers fail :)

Reply:
Only that currently derived classes don't really do anything in the constructor (I've started doing a bit more in sublimelsp/lsp_utils#5 but still not so much that it would be likely to crash). Is it worth it really?

Reply:
Well OK, I've done a bit more that could crash.